testing/test_collection.py

```diff
@@ -1789,6 +1789,26 @@ def test_collect_short_file_windows(pytester: Pytester) -> None:
     assert result.parseoutcomes() == {"passed": 1}
 
 
+def test_not_collect_symlink_syblings(
+    pytester: Pytester, tmp_path: Path, request: pytest.FixtureRequest
```

Comment:
Thanks @nicoddemus, that's much better! Nitpicking: its "siblings", not "syblings".

Reply:
Thanks!